content/docs/dvclive/user-guide/ml-frameworks/keras.md

```diff
@@ -0,0 +1,71 @@
+# Keras
+
+The DVCLive - Keras integration allows you to easily add experiment tracking
+capabilities to your Keras projects.
+
+The integration is
+[maintained in the DVCLive repository](https://github.com/iterative/dvclive/blob/master/dvclive/keras.py)
+
+## About Keras
+
+[Keras](https://keras.io/) is a central part of the tightly-connected TensorFlow
+2.0 ecosystem, covering every step of the machine learning workflow, from data
+management to hyperparameter training to deployment solutions.
+
+## Usage
+
+To start using the integration you just need to add a few lines to your training
+code in **any** Keras project.
+
+You just need to add the `DvcLiveCallback` to the callbacks list passed to your
+`model`:
+
+```git
++from dvclive.keras import DvcLiveCallback
+
+. . .
+
+model.fit(
+    train_dataset,
+    epochs=num_epochs,
+-    validation_data=validation_dataset)
++    validation_data=validation_dataset,
++    callbacks=[DvcLiveCallback()])
+```
+
+This will generate the metrics logs and summaries as described in the
+[Quickstart](/docs/dvclive/user-guide/quickstart#outputs).
+
+> 💡Without requiring additional modifications to your training code, you can
+> use the DVCLive - Keras integration alongside DVC. See
+> [DVCLive with DVC](/doc/dvclive/user-guide/dvclive-with-dvc) for more info.
+
+## Parameters
+
+- `model_file` - The name of the file where the model will be saved at the end
+  of each `step`.
+
+- `save_weights_only` (`False` by default) - if True, then only the model's
+  weights will be saved (`model.save_weights(model_file)`), else the full model
+  is saved (`model.save(model_file)`)
+
+Example:
+
+```python
+from dvclive.keras import DvcLiveCallback
+
+model.fit(
+    train_dataset,
+    epochs=num_epochs,
+    validation_data=validation_dataset,
+    callbacks=[DvcLiveCallback(
+        model_file="my_model_weights.h5",
+        save_weights_only=True)])
+```
+
+## Example repository
+
+You can find a fully working example using the DVCLive - Keras integration in
+the following link:
+
+https://github.com/iterative/example-ml-frameworks/tree/keras
```

Comment:
Is this the only framework for this example repo right now? I thought you had other branches?

Reply:
I had. I forgot to comment on this. For each ML Framework I had an example using dvclive implemented separately which felt a little inconsistent. 

I though it would me more useful to have a "main" branch implemented in one ML Framework (i.e. keras) and then create the other examples using that branch as starting point, which is a work in progress I would probably finish tomorrow.

The main idea is to show consistency across all the dvc pipeline with the exception of the ML Framework specific code, kind of explicitly showcasing that `dvclive` is ML Framework agnostic and you would get very similar features with very small modifications.

Reply:
Sounds good. Maybe it's better to leave this out for now and put all of that in a separate PR?

Reply:
Yep that's why I (think) I removed every link except for keras

Reply:
Or did you mean all of the ML Framework stuff?

Reply:
No, I was suggesting you might also want to remove the keras link, but you can leave as is as long as you don't expect the keras link to change.